.nuspec/Microsoft.Maui.Resizetizer.targets

```diff
@@ -671,6 +676,36 @@
         <ItemGroup>
             <FileWrites Include="$(_MauiManifestStampFile)" />
         </ItemGroup>
+
+    </Target>
+
+    <Target Name="_MauiManifestMergerAfterTarget"
+            AfterTargets="_ManifestMerger"
+            Condition=" '$(_ResizetizerIsAndroidApp)' == 'True' And '$(AndroidManifestMerger)' == 'manifestmerger.jar' "
+            DependsOnTargets="_MauiGenerateAndroidManifestAttributes" />
+
+    <Target Name="_MauiGenerateJavaStubsAfterTarget"
+            AfterTargets="_GenerateJavaStubs"
+            Condition=" '$(_ResizetizerIsAndroidApp)' == 'True' And '$(AndroidManifestMerger)' != 'manifestmerger.jar' "
+            DependsOnTargets="_MauiGenerateAndroidManifestAttributes" />
+
+    <Target Name="_MauiGenerateAndroidManifestAttributes"
+            Inputs="$(MSBuildThisFileFullPath);$(_ResizetizerTaskAssemblyName);$(_ResizetizerInputsFile);$(_MauiSplashInputsFile);$(IntermediateOutputPath)android\AndroidManifest.xml"
+            Outputs="$(_MauiManifestStampFile);$(IntermediateOutputPath)android\AndroidManifest.xml"
+            Condition=" '$(_ResizetizerIsAndroidApp)' == 'True' ">
+
+        <GenerateAndroidManifestXml
+            AppIcon="@(MauiImage->WithMetadataValue('IsAppIcon', 'true'))"
+            Manifest="$(IntermediateOutputPath)android\AndroidManifest.xml"
+            IntermediateOutputPath="$(IntermediateOutputPath)android"
+            GeneratedFilename="AndroidManifest.xml" />
```

Comment:
I'd prefer to put this in the Android workload? If this is only for `<application android:icon="" android:roundIcon="" />`, can we just add MSBuild properties for both of these?

I'll review this PR more this afternoon, thanks.

Reply:
This PR is quite large (647 files changed), maybe you can take this specific part out for a separate PR?

If we introduce new MSBuild properties for `icon`/`roundIcon`, that is something that would be good for .NET 7. I wouldn't want to add `AndroidManifest.xml` generation in MAUI at all for this, as it could easily hurt incremental build times if it's done wrong. That also makes me question if it should be in a service release.

Reply:
OK, good point. Let me move this new feature out into a new place.

Reply:
Moved to https://github.com/dotnet/maui/pull/8063